static/js/loader.js

```diff
@@ -1,7 +1,7 @@
 var script = document.createElement('script');
 
 script.src = "https://media.evolv.ai/asset-manager/releases/next/webloader.js";
-script.setAttribute('data-evolv-environment', "cecc5e6e80");
-script.setAttribute('data-evolv-endpoint', "https://participants-newdev.evolvdev.com/");
-script.setAttribute('data-evolv-uid', "45389200_1626264858193");
+script.setAttribute('data-evolv-environment', "765b631737");
```

Comment:
This should be deleted, and added to gitugnore so we don't get confsed that we need to configure ids here